geekhours/test/test_command.py

```diff
@@ -158,3 +158,39 @@ def test_dump_to_json(self):
 
         with self.assertRaises(FileExistsError):
             self._command.dump_to_json(records, jsonfile)
+
+    def test_show_total_hours(self):
+        """ Test show_total_hours()
+
+        Assert show_total_hours() calls Database.get_total_hours() and
+        succeeds.
+        """
+        self._command.show_total_hours()
```

Comment:
Why no assertion here? test_command.py does not use test_database.py. Hence, you have to assert the return value of the functions even assertion is similar to the test_database.py did.

Reply:
Fixed it to use assertion.